smallBamQc.wdl

```diff
@@ -122,12 +122,24 @@ workflow smallBamQc {
                 url : "https://bedtools.readthedocs.io"
             }
         ]
-        output_meta: {
-            samtools: "Samtools stats output",
-            picard: "Picard MarkDuplicates output",
-            bedtoolsCoverage: "Bedtools coverage histogram output",
-            downsampledCounts: "JSON file recording what downsampling was done"
-        }
+    output_meta: {
+    samtools: {
```

Comment:
should samtools and picard labels be more descriptive, ie. "samtoolsMetrics" and "picardMarkDuplicateMetrics"? or is that too wordy?

Reply:
Not sure, it looks fine to me. I am not adding labels, just re-formatting.